lib/hyrax/transactions/steps/set_uploaded_date_unless_present.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+module Hyrax
+  module Transactions
+    module Steps
+      ##
+      # A step that sets the uploaded date to now for an input resource or
```

Comment:
Based on the `date_modified` method I believe the documentation could be tightened a bit. Perhaps an `@see`? Given the specificity of these classes, I can see further documenting the specific implementation details.

Reply:
thanks i think this can be expanded pretty easily and will push up some changes.